tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,6 +14,7 @@
 class Homescreen(Base):
 
     name = 'Homescreen'
+    origin = 'app://verticalhome.gaiamobile.org'
```

Comment:
Nit: to be removed once [bug 1188926](https://bugzilla.mozilla.org/show_bug.cgi?id=1188926) lands